app/electron/__tests__/test-data/artifacthub-pkg.yml

```diff
@@ -0,0 +1,32 @@
+version: 0.1.0
+name: headlamp_minikube
+displayName: Minikube
+createdAt: '2024-11-08T10:00:00Z'
+logoURL: 'https://minikube.sigs.k8s.io/images/logo/logo.png'
+description: A Headlamp plugin for managing Minikube clusters
+annotations:
+  headlamp/plugin/archive-url: 'https://github.com/headlamp-k8s/plugins/releases/download/example-change-logo-0.0.1/change-logo-0.0.1.tar.gz'
+  headlamp/plugin/archive-checksum: 'SHA256:8a2a23d116fe175b4350ffc6cc259781ed98d943f932e30fc5c8fe4e07b86e89'
+  headlamp/plugin/version-compat: '>=0.30'
+  headlamp/plugin/distro-compat: 'in-cluster,web,docker-desktop,desktop'
+
+  # Platform-specific archives contain the minikube binary for each platform
+  # Linux x64
+  headlamp/plugin/archive-url:linux:x64: 'https://storage.googleapis.com/minikube/releases/latest/minikube-linux-amd64.tar.gz'
```

Comment:
"https://storage.googleapis.com/minikube" should be added to allowed archive urls?

Reply:
You are right. However, I think the tests were checking this and passing, so now I am puzzled.

Reply:
Let's allow-list the URLs once we have a real use case for them. Another option is to allow all of them, since we are showing them to the user...

Reply:
Ok, so this isn't needed after all.

From the minikube installation documentation they serve their binaries from github releases:
> https://github.com/kubernetes/minikube/releases/latest/download/minikube-windows-amd64.exe